AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -168,13 +167,26 @@ + (void)trackEvent:(NSString *)eventName withProperties:(nullable NSDictionary<N
   [self trackEvent:eventName withProperties:properties forTransmissionTarget:nil];
 }
 
-+ (void)   trackEvent:(NSString *)eventName
-       withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties
-forTransmissionTarget:(nullable MSAnalyticsTransmissionTarget *)transmissionTarget {
-  @synchronized (self) {
++ (void)trackEvent:(NSString *)eventName withTypedProperties:(nullable MSEventProperties *)properties {
```

Comment:
Might want to check the formatting on this.  I would have expected the `withTypedProperties` parameter to be on a new line.